src/Microsoft.Build.Sql/README.md

```diff
@@ -7,36 +7,28 @@
 This project is in its early stages and we are currently building tests for different functionality. Contributors welcome, please feel free to share feedback or PRs.
 
 ## Using this SDK
-Normally the SDK can be pulled from public Nuget feeds. However since we are still in development mode, this SDK can only be consumed from a local Nuget source. [See also](#adding-a-local-nuget-source)
 
-### Building this project
-1. Build this project by running `dotnet build` from `\src\Microsoft.Build.Sql\`. This will generate a Nuget package inside the `bin` folder.
-2. Copy the nupkg file to a well known location, e.g. `C:\local_packages\`
-
-### Adding a local Nuget source
-1. Add a `nuget.config` in the same folder as your sqlproj by running 
-   ```
-   dotnet new nugetconfig
-   ```
-2. Edit `nuget.config` by adding a package source pointing to the folder that contains Microsoft.Build.Sql.1.0.0.nupkg
-   ```xml
-    <packageSources>
-      <clear />
-      <add key="local" value="{PATH_OF_SDK}" />
-    </packageSources>
-   ```
+### Add Github Packages as Nuget Source
```

Comment:
for easier use of links by folks who are not English-language dominant, it's best practice to see if a docs link auto-resolves if you remove the language-specific segment.
eg: https://docs.github.com/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry#authenticating-to-github-packages

nit: GitHub with capital H